lib/register.js

```diff
@@ -17,7 +17,10 @@ const proxyHandler = {
     if (name === Symbol.toStringTag) {
       return 'Module'
     }
-    return getters.get(target)[name]()
+
+    if (typeof getters.get(target)[name] === 'function') {
```

Comment:
Could you change this so it only calls `getters.get` once?

Reply:
[90d0930](https://github.com/nodejs/import-in-the-middle/pull/155/commits/90d093066d23dd1fed0abc8e98a0e761a92599ca)